mote/.eslintrc

```diff
@@ -1,31 +1,57 @@
 {
-    "ecmaFeatures": {
-        "jsx": true
+    "parser": "babel-eslint",
+
+    "plugins": [],
+
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "globalReturn": true,
+            "jsx": true,
+            "modules": true,
+        }
     },
+
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true,
+        "commonjs": true
+    },
+
+    "globals": {
+        "document": true,
+        "escape": true,
+        "navigator": true,
+        "unescape": true,
+        "window": true,
+        "describe": true,
+        "before": true,
+        "it": true,
+        "expect": true,
+        "sinon": true
+    },
+
     "rules": {
-        "quotes": [1, "single"],
-        "strict": [2, "global"],
-        "require-jsdoc": [1],
-        "valid-jsdoc": [2],
-        "block-spacing": [2],
-        "brace-style": [2, "1tbs", { "allowSingleLine": false }],
-        "camelcase": [2, { "properties": "always" }],
-        "block-spacing": [2, "always"],
-        "comma-style": [2, "last"],
-        "comma-spacing": [2, { "before": false, "after": true }],
-        "indent": [2, 4],
+        "quotes": ["error", "single"],
+        "strict": ["error", "global"],
+        "require-jsdoc": [0],
+        "valid-jsdoc": [0],
+        "brace-style": ["error", "1tbs", { "allowSingleLine": false }],
+        "camelcase": ["error", { "properties": "always" }],
+        "block-spacing": ["error", "always"],
+        "comma-style": ["error", "last"],
+        "comma-spacing": ["error", { "before": false, "after": true }],
+        "indent": ["error", 4, {"SwitchCase": 1}],
         "no-ternary": [0],
-        "quote-props": [2, "always"],
-        "space-before-function-paren": [2, "never"],
-        "padded-blocks": [2, "never"],
-        "semi-spacing": [2],
-        "space-infix-ops": [2],
-        "space-before-keywords": [2, "always"],
-        "space-in-parens": [2, "never"],
-        "no-mixed-spaces-and-tabs": [2]
-    },
-    "env": {
-        "amd": true,
-        "es6": true
+        "padded-blocks": ["error", "never"],
```

Comment:
Sure. This is best done directly in the mobius-skeleton repo, however, as that's our source of truth for all our dotfiles now.